### ISSUE TITLE
[Enahncement] PXC File JSON Minification

### ISSUE THREAD

**Issue by Synthoid** [Ttanasart-pt/Pixel-Composer#30]
Currently the JSON for a `.pxc` file is fairly minified, but it does leave some spaces that can be removed to reduce file size even more.

***Empty Project (Current)***
```
{ "nodes": [ ], "graph": { "graph_x": 560.0, "graph_y": 338.0 }, "version": 1100.0, "animator": { "framerate": 8.0, "frames_total": 16.0 } }
```

***Empty Project (Minified)***
```
{"nodes":[],"graph":{"graph_x":560.0,"graph_y":338.0},"version":1100.0,"animator":{"framerate":8.0,"frames_total":16.0}}
```

This minor data optimization could probably be implemented whenever since it shouldn't affect functionality, and would help shave off some KB as files get larger and larger.

**Comment by Ttanasart-pt** [comment]
Sure, currently I just use default GameMaker json function. I think I can optimize save file a lot more (also deal with those weirdly long decimal value e.g. 0.10000000000000000555111512312578)

**Comment by Ttanasart-pt** [comment]
Implemented in 1.12
